assets/docs.html

```diff
@@ -150,33 +147,34 @@ <h2 class="docs-subtitle">Contributors</h2>
 					<div class="span4">
 						<div class="footer-title">ABOUT US</div>
 						WindBot is a high quality automation and enhancement software that functions alongside the Tibia client. It works by sending mouse clicks and keyboard strokes, displaying it's own visuals on top, facilitating player communication and is customizable in many many ways.
-						<br><br>
+						<br /><br />
 						It was created by Lucas Terra, one of the main minds behind the success of NeoBot and ElfBot. It took several months of work and tests before...
-						<br><br>
+						<br /><br />
 						<a href="about-us.html" class="blue">Read more »</a>
 					</div>
 					<div class="span4">
 						<!-- Sections -->
 						<div class="footer-title">SECTIONS</div>
-						<a href="news.html">» News</a><br><br>
-						<a href="features.html">» Features</a><br><br>
-						<a href="download.html">» Download</a><br><br>
-						<a href="purchase.html">» Purchase</a><br><br>
-						<a href="lua-variables.html">» Documentation</a><br><br>
+						<a href="news.html">» News</a><br /><br />
+						<a href="features.html">» Features</a><br /><br />
+						<a href="download.html">» Download</a><br /><br />
+						<a href="purchase.html">» Purchase</a><br /><br />
+						<a href="lua-variables.html">» Documentation</a><br /><br />
```

Comment:
@ lib files

OK I'll introduce PR with them separately. I was afraid that you'll rice concerns about that one but for security point of view. In theory it's hard to verify lib files (I could put there whatever I want minified).

You say it's better for maintaining I say final goal would be to get rid of them completely in favor for dependencies manager.

Other solution is to rely on official CDNs.

Agains that is that one big file downloads faster than multiple small files and either way we should have concater+minifier.

Can you reflect on that?